public/app/plugins/datasource/loki/querybuilder/components/LokiQueryBuilder.tsx

```diff
@@ -35,6 +45,10 @@ export interface Props {
 export const LokiQueryBuilder = React.memo<Props>(({ datasource, query, onChange, onRunQuery, showExplain }) => {
   const [sampleData, setSampleData] = useState<PanelData>();
   const [highlightedOp, setHighlightedOp] = useState<QueryBuilderOperation | undefined>(undefined);
+  const [conflictingLabels, setConflictingLabels] = useState(false);
+  const [labels, setLabels] = useState<Array<Partial<QueryBuilderLabelFilter>>>([]);
```

Comment:
I don't understand why do we need this duplicated as we already have labels from props.query.labels?